lib/wit.js

```diff
@@ -210,7 +210,10 @@ const validate = (opts) => {
   if (opts.actions) {
     opts.actions = validateActions(opts.logger, opts.actions);
   }
-
+  if(!opts.maxSteps || !Number.isInteger(opts.maxSteps)) {
```

Comment:
this is not necessary, as the maxSteps in this.config is already enforced into being a number here -> 

https://github.com/num8er/node-wit/blob/363a23dccc5f04eb6068ce98e171c1db256989ca/lib/wit.js#L213